preparing-to-code/environment-setup/README.md

```diff
@@ -31,7 +31,7 @@ The purpose of these notes is to help you install and configure the software we
 * [Google Chrome (web browser)](https://www.google.com/chrome/browser/desktop/index.html)
 * [Atom (text editor)](https://atom.io/)
 * [Ruby Version Manager (rvm) (package manager)](https://rvm.io/)
-* [Ruby 2.3.0](https://www.ruby-lang.org/en/)
+* [Ruby 2.4.1](https://www.ruby-lang.org/en/)
```

Comment:
Can we just list "latest version of Ruby" here to we don't have to continue to update it?

Reply:
Updated for this line. 
Just as note: https://github.com/Ada-Developers-Academy/jump-start/tree/master/preparing-to-code/environment-setup#ruby will need to be kept up to date. I'll check it at the beginning of every JSL.